Source/interfaces/IPlayerInfo.h

```diff
@@ -78,5 +78,27 @@ namespace Exchange {
         virtual IAudioIterator* AudioCodec() const = 0;
         virtual IVideoIterator* VideoCodec() const = 0;
     };
+
+    struct EXTERNAL IPlayerAtmosProperties : virtual public Core::IUnknown {
```

Comment:
Atmos is part of the Dolby eco system. Sugegst to put this in the Dolby ecosystem:
https://github.com/rdkcentral/Thunder/blob/master/Source/interfaces/IDolby.h
I think there you will find that the setAudioAtmosOutputMode is already more specific defined.

Also our coding rules require methods to be PascalCase and not CamelCase. If you do not have the coding Rules, let me know, I can share them.

Reply:
Currently we are adding the IVideo Atmos functionality as a Dolby feature. Good to align maybe so these two would work together. See pull request: https://github.com/rdkcentral/Thunder/pull/212 but also there the remark to move to the dolby space :-)

Reply:
The concept of the IUnknown (comrpc) allows for interface navigation. Through interface navigation you can "check" if certain features are available or not (you get the interface or you do not get it) SO there is no need to create another plugin. The PlayerInfo plugin and the DisplayInfo plugin will just by extended with IDolby*. As you can see these plugins have a sub folder where the implementation resides. As discusses , you can create an IARM implementation (since you deliegate the retrieval of this information from yet another component). 
If for your platofrm you do not support the Dolby, it just means that the IDiolby interface return null and there is no implementation for that platform (e.g. RPI will not have an IDolby implementation) 
So I think the interface navigation is already handled by the base system